src/js/jsx/shared/Draggable.jsx

```diff
@@ -99,71 +117,77 @@ define(function (require, exports, module) {
                 return {
                     // Whether or not currently dragging
                     dragging: false,
-
-                    // Start top/left of the DOM node
-                    startX: null, startY: null,
-                    dragStyle: null
+                    initialDragPosition: null
                 };
             },
+            
+            // Instead of responding to DragAndDrop's all change events, it listens to the "start-drag" event (fires 
```

Comment:
For consistency, methods and functions should be prefixed with block-style comments instead of line-style comments. Also, please mark it as `@private`.